### ISSUE TITLE
client.pricing.sms.list gives 404 response

### ISSUE THREAD

**Issue by Yaroslav-F** [Vonage/vonage-ruby-sdk#96]
As simple as it says in the title:
```
irb(main):020:0> client.pricing.sms.list
Traceback (most recent call last):
1: from (irb):20
Nexmo::ClientError (404 response from rest.nexmo.com)
```

Nexmo client is valid.

What can I be missing?

**Comment by timcraft** [comment]
@Yaroslav-F Not missing anything, that's a bug. Looks like the endpoints are incorrect.

**Comment by Yaroslav-F** [comment]
Thanks Tim.
I was thinking on fixing the endpoint myself but just couldn't find one myself in the [api docs](https://developer.nexmo.com/api/developer/pricing). 😕

**Comment by timcraft** [comment]
@Yaroslav-F No worries, leave it with me. I’ll get it fixed as soon as I get chance and push out a 5.0.2 release.

**Comment by timcraft** [comment]
@Yaroslav-F Fixed in 8c2e653 (available in v5.0.2). Thanks for the heads up!
